src/analyzer/analyzerwaveform.cpp

```diff
@@ -75,6 +82,11 @@ bool AnalyzerWaveform::initialize(TrackPointer tio, int sampleRate, int totalSam
         m_waveformSummary = WaveformPointer(new Waveform(
                 sampleRate, totalSamples, mainWaveformSampleRate,
                 summaryWaveformSamples));
+        //I force it to NOT dirty, because other Mixxx threads like the Main thread can trigger 
```

Comment:
No strong need for a change, but we use a space after // to distinguish disabled code from comments. 
```
// comment
//code   
```